[2023-12-27T00:02:31.183+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_filtered_customers_table scheduled__2023-12-26T00:00:00+00:00 [queued]>
[2023-12-27T00:02:31.191+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_filtered_customers_table scheduled__2023-12-26T00:00:00+00:00 [queued]>
[2023-12-27T00:02:31.193+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T00:02:31.195+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-12-27T00:02:31.197+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T00:02:31.211+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_filtered_customers_table> on 2023-12-26 00:00:00+00:00
[2023-12-27T00:02:31.216+0000] {standard_task_runner.py:55} INFO - Started process 22708 to run task
[2023-12-27T00:02:31.220+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'potential_customers_process', 'create_filtered_customers_table', 'scheduled__2023-12-26T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/potential_customers_process.py', '--cfg-path', '/tmp/tmp1r8sh3ix']
[2023-12-27T00:02:31.221+0000] {standard_task_runner.py:83} INFO - Job 9: Subtask create_filtered_customers_table
[2023-12-27T00:02:31.273+0000] {task_command.py:376} INFO - Running <TaskInstance: potential_customers_process.create_filtered_customers_table scheduled__2023-12-26T00:00:00+00:00 [running]> on host 886ab54d5808
[2023-12-27T00:02:31.318+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=potential_customers_process
AIRFLOW_CTX_TASK_ID=create_filtered_customers_table
AIRFLOW_CTX_EXECUTION_DATE=2023-12-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-26T00:00:00+00:00
[2023-12-27T00:02:31.327+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-12-27T00:02:31.332+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS filtered_customers (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                gender TEXT NOT NULL,
                age INT NOT NULL,
                last_login TEXT NOT NULL,
                external_id TEXT NOT NULL
            );
        , parameters: None
[2023-12-27T00:02:31.334+0000] {postgres.py:96} INFO - NOTICE:  relation "filtered_customers" already exists, skipping

[2023-12-27T00:02:31.341+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=potential_customers_process, task_id=create_filtered_customers_table, execution_date=20231226T000000, start_date=20231227T000231, end_date=20231227T000231
[2023-12-27T00:02:31.393+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-27T00:02:31.414+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
